src/hiveboard_bridge/src/main.cpp

```diff
@@ -0,0 +1,34 @@
+#include "../include/hiveboard_bridge/TCPServer.h"
+#include "../include/hiveboard_bridge/TCPServerMonitor.h"
+#include "ros/ros.h"
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "hiveboard_bridge");
+
+    TCPServerMonitor monitor;
+    TCPServer socket(8080, monitor);
+
+    ROS_INFO("Listening for incoming clients...");
+    socket.listen();
+    ROS_INFO("Client connected.");
+
+    char sendValue[] = "Hello world from ROS!";
```

Comment:
Will we always keep that message?

Reply:
The higher layers to manage the communication will be added later. Right now this simply serves as an example (and for testing purposes since there are no automated tests for this driver).